From 5f17907576df923557f0bb1904d2ea8ee4484eb5 Mon Sep 17 00:00:00 2001
From: Tim Snyder <snyder.tim@gmail.com>
Date: Wed, 15 Mar 2023 13:47:37 +0000
Subject: [PATCH] rm project-specified default std for C++

This should enable cmake to not pass a -std switch and that's what we want
the default used by our build toolchain
---
 CMakeLists.txt | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 91a901730..ab634f6b2 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -277,12 +277,6 @@ if (NOT CMAKE_BUILD_TYPE)
        FORCE)
 endif (NOT CMAKE_BUILD_TYPE)
 
-set (CMAKE_CXX_STANDARD 98 CACHE STRING
-     "C++ standard to build all targets. Supported values are 98, 11, 14, and 17.")
-set (CMAKE_CXX_STANDARD_REQUIRED ON CACHE BOOL
-     "The with CMAKE_CXX_STANDARD selected C++ standard is a requirement.")
-mark_as_advanced (CMAKE_CXX_STANDARD_REQUIRED)
-
 if (NOT (WIN32 OR CYGWIN))
   option (BUILD_SHARED_LIBS "Build shared libraries." ON)
 else (NOT (WIN32 OR CYGWIN))
-- 
2.39.1

